core/src/commonMain/kotlin/ua/syt0r/kanji/presentation/common/resources/string/JapaneseStrings.kt

```diff
@@ -141,75 +141,75 @@ object JapaneseStatsStrings : StatsStrings {
-    override val inputHint: String = "文字や単語を入力"
+    override val inputHint: String = "文字または単語を入力"
     override val charactersTitle: (count: Int) -> String = { "文字 ($it)" }
     override val wordsTitle: (count: Int) -> String = { "単語 ($it)" }
     override val radicalsSheetTitle: String = "部首で検索"
     override val radicalsFoundCharacters: String = "見つかった文字"
-    override val radicalsEmptyFoundCharacters: String = "何も見つかりません"
+    override val radicalsEmptyFoundCharacters: String = "何も見つかりませんでした"
     override val radicalSheetRadicalsSectionTitle: String = "部首"
 }
 
 object JapaneseAlternativeDialogStrings : AlternativeDialogStrings {
     override val title: String = "別の単語"
     override val readingsTitle: String = "読み方"
     override val meaningsTitle: String = "意味"
-    override val button: String = "キャンセル"
+    override val button: String = "閉じる"
 }
 
 object JapaneseSettingsStrings : SettingsStrings {
     override val analyticsTitle: String = "分析レポート"
     override val analyticsMessage: String = "アプリを向上させるために匿名データの送信を許可する"
     override val themeTitle: String = "テーマ"
-    override val themeSystem: String = "システム"
+    override val themeSystem: String = "システムに従う"
     override val themeLight: String = "ライト"
     override val themeDark: String = "ダーク"
     override val reminderTitle: String = "リマインダー通知"
     override val reminderEnabled: String = "有効"
     override val reminderDisabled: String = "無効"
-    override val aboutTitle: String = "アプリについて"
+    override val aboutTitle: String = "このアプリについて"
 }
 
 object JapaneseReminderDialogStrings : ReminderDialogStrings {
     override val title: String = "リマインダー通知"
-    override val noPermissionLabel: String = "通知の許可がない"
-    override val noPermissionButton: String = "許可を付与する"
+    override val noPermissionLabel: String = "通知の権限がありません"
+    override val noPermissionButton: String = "許可する"
     override val enabledLabel: String = "通知"
     override val timeLabel: String = "時間"
     override val cancelButton: String = "キャンセル"
     override val applyButton: String = "適用"
 }
 
 object JapaneseAboutStrings : AboutStrings by EnglishAboutStrings {
-    override val title: String = "アプリについて"
+    override val title: String = "このアプリについて"
     override val version: (versionName: String) -> String = { "バージョン: $it" }
 }
 
 object JapanesePracticeImportStrings : PracticeImportStrings {
 
-    override val title: String = "選ぶ"
+    override val title: String = "選択"
 
-    override val kanaTitle: String = "かな"
+    override val kanaTitle: String = "仮名"
 
     override val kanaDescription = { urlColor: Color ->
         buildAnnotatedString {
-            append("いちばんかんたんな日本語の文字。かなには二つ部分があります:\n")
-            append("•　平仮名（ひらがな）ー　日本語のたんごやおとを伝えに使う\n")
-            append("•　片仮名（かたかな）ー　外国語のたんごを書くときに使う")
+            append("仮名は、最も簡単な日本語の文字です。仮名は二種類に分けることができます。\n")
+            append("・平仮名（ひらがな）─ 日本語の単語や音を伝えるときに使う\n")
+            append("・片仮名（かたかな）─ 主に外来語の単語を書くときに使う")
```

Comment:
First of all thank you for reviewing my translations and these fixes! Are you native or advanced speaker?)

Regarding this place though, I tried to word it here as simple as possible and avoid using kanji too much so that a potential child could understand what is going on :sweat_smile: 

It's probably not realistic that Japanese children will use my app because it only has English translations right now and the app is quite complicated overall, but if possible I would like to keep it worded very simple. What do you think?

Also, I'm not sure if I'll do it, but I want to try generating kanji/words definitions in Japanese with AI, in addition to adding kana voices, I already have several generated voice lines ready

Reply:
> Are you native or advanced speaker?

I'm a native speaker.
> I would like to keep it worded very simple. What do you think?

Makes sense. I tried rewriting in [Easy Japanese](https://en.wikipedia.org/wiki/Easy_Japanese) and with up to Grade 2 of kanji usage.